app/src/main/java/com/netguru/codereview/MainActivity.kt

```diff
@@ -1,11 +1,14 @@
 package com.netguru.codereview
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.netguru.codereview.ui.MainFragment
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        supportFragmentManager.beginTransaction().replace(R.id.container, MainFragment()).commitNow()
```

Comment:
Keep in mind that `commit` should be used if multiple fragments are to be used and usage of FragmentManager's back stack may be required, as `commitNow` does not add transaction to it.